solutions/swb-reference/src/services/sshKeyService.test.ts

```diff
@@ -497,13 +502,374 @@ describe('SshKeyService', () => {
     let sendPublicKeyRequest: SendPublicKeyRequest;
 
     beforeEach(() => {
-      sendPublicKeyRequest = { environmentId: '' };
+      sendPublicKeyRequest = {
+        environmentId: 'env-123',
+        userId: '1234'
+      };
+    });
+
+    describe('when environment does not exist', () => {
+      beforeEach(() => {
+        environmentService.getEnvironment = jest.fn(() => {
+          throw new Error(`Could not find environment ${sendPublicKeyRequest.environmentId}`);
```

Comment:
Do we have a custom error for this scenario?

Reply:
No because Environment service is still throwing boom errors